web/src/utils.js

```diff
@@ -1,3 +1,5 @@
+const rootDandiUrl = 'dandiarchive.org';
```

Comment:
While I'm sure it'll always be https, let's change this to be `https://dandiarchive.org`.

Reply:
The reason I originally did it this way was to allow for subdomains like `hub.dandiarchive.org` and `girder.dandiarchive.org`, but I've updated it.

Reply:
If we get subdomains, I'd say those should just be entirely separate variables. This is the sort of thing that I'd normally use `process.env` for so that the application is agnostic (to whatever extent possible) to its deployed environment.